assets/js/modules/sign-in-with-google/components/settings/SettingsView.js

```diff
@@ -0,0 +1,218 @@
+				<div className="googlesitekit-settings-module__meta-item">
+					<h5 className="googlesitekit-settings-module__meta-item-type">
+						{ __( 'Client ID', 'google-site-kit' ) }
+					</h5>
+					<p className="googlesitekit-settings-module__meta-item-data">
+						<DisplaySetting value={ clientID } />
+					</p>
+				</div>
+			</div>
+
+			<div className="googlesitekit-settings-module__meta-items">
+				{ !! buttonTextLabel && (
+					<div className="googlesitekit-settings-module__meta-item">
+						<h5 className="googlesitekit-settings-module__meta-item-type">
+							{ __( 'Button text', 'google-site-kit' ) }
+						</h5>
+						<p className="googlesitekit-settings-module__meta-item-data">
+							<DisplaySetting value={ buttonTextLabel } />
+						</p>
+					</div>
+				) }
+
+				{ !! buttonThemeLabel && (
+					<div className="googlesitekit-settings-module__meta-item">
+						<h5 className="googlesitekit-settings-module__meta-item-type">
+							{ __( 'Button theme', 'google-site-kit' ) }
+						</h5>
+						<p className="googlesitekit-settings-module__meta-item-data">
+							<DisplaySetting value={ buttonThemeLabel } />
+						</p>
+					</div>
+				) }
+
+				{ !! buttonShapeLabel && (
+					<div className="googlesitekit-settings-module__meta-item">
+						<h5 className="googlesitekit-settings-module__meta-item-type">
+							{ __( 'Button shape', 'google-site-kit' ) }
+						</h5>
+						<p className="googlesitekit-settings-module__meta-item-data">
+							<DisplaySetting value={ buttonShapeLabel } />
+						</p>
+					</div>
+				) }
+			</div>
+
+			<div className="googlesitekit-settings-module__meta-items">
+				<div className="googlesitekit-settings-module__meta-item">
+					<h5 className="googlesitekit-settings-module__meta-item-type">
+						{ __( 'One tap sign in', 'google-site-kit' ) }
+					</h5>
+					<p className="googlesitekit-settings-module__meta-item-data">
+						<DisplaySetting
+							value={
+								oneTapEnabled
+									? __( 'Enabled', 'google-site-kit' )
+									: __( 'Disabled', 'google-site-kit' )
+							}
+						/>
+					</p>
+				</div>
+			</div>
+
+			<div className="googlesitekit-settings-module__meta-items">
+				<div className="googlesitekit-settings-module__meta-item">
+					<h5 className="googlesitekit-settings-module__meta-item-type">
+						{ __(
+							'Users can create new accounts',
+							'google-site-kit'
+						) }
+					</h5>
+					{ anyoneCanRegister !== undefined && (
+						<p className="googlesitekit-settings-module__meta-item-data">
+							<DisplaySetting
+								value={
+									anyoneCanRegister
+										? __( 'Enabled', 'google-site-kit' )
+										: __( 'Disabled', 'google-site-kit' )
+								}
+							/>
+						</p>
+					) }
+
+					{ anyoneCanRegisterNoticeDismissed !== undefined &&
+						anyoneCanRegisterNoticeDismissed !== true &&
+						anyoneCanRegister === false && (
+							<SettingsNotice
+								type={ TYPE_WARNING }
```

Comment:
Not sure what happened here, but this looks quite different from [the design](https://www.figma.com/design/h9yGR15Pg2DUtgERW4AFAv/Sign-in-with-Google?node-id=248-1545&t=oNrLlxQH5cE1QHP7-4)

<img width="997" alt="image" src="https://github.com/user-attachments/assets/83301117-09b2-4f13-a7c1-bcc392c454ae">

Reply:
I just used the `Notice` component so I figured it was a mismatch between the components and Figma, but I can make some adjustments 🤔

Reply:
I'm guessing we weren't really using it elsewhere, but there are a few instances. A bit concerning that it looks so off and we haven't noticed it elsewhere 🤔

Reply:
> My instinct is always to use our base components as they are, and have issues to modify them, but this change was small so I made it here.

Same! Let's open a follow-up issue to make sure our base components match the current designs so that we avoid introducing inconsistency.